src/DIRAC/Core/Security/m2crypto/X509Chain.py

```diff
@@ -976,6 +976,13 @@ def getCredentials(self, ignoreDefault=False, withRegistryInfo=True):
             if retVal["OK"]:
                 credDict["username"] = retVal["Value"]
                 credDict["validDN"] = True
+
+            retVal = self.getDIRACGroup(ignoreDefault=ignoreDefault)
+            if retVal["OK"]:
+                diracGroup = retVal["Value"]
+                credDict["group"] = diracGroup
```

Comment:
No ! This should be `"hosts"`, see above

Reply:
This thing is insanely fragile, and the fact that the tests pass shows how bad we are  !
The assumption that "hosts" is a special case is done in quite a few places. Fortunately, if we need to change that, it would all be server side, and synchronous (no proxies stored for ever). So we could think of transitioning if needed. The question is: what do we need, to accomplish what ?

Reply:
Ah, you are right. I removed this change.